app/models/email_configuration.rb

```diff
@@ -46,9 +46,9 @@ def only_auth_changes?
 
   protected
 
-  def set_tenant_id
-    self.tenant_id ||= account&.id if account&.provider? && !account&.master?
-  end
+  # def set_tenant_id
+  #   self.tenant_id ||= account&.id if account&.provider? && !account&.master?
+  # end
```

Comment:
yeah, it can be done in another PR or just cleaned up here. Just don't want to forget taking care of it. btw there are no tests to first see if this approach we will take.